File: array/CHAR INDEX.py
//CHAR INDEX

class Solution:
    # @param A : string
    # @return a strings
    def solve(self, A):
        alpha = ('abcdefghijklmnopqrstuvwxyz')
        b=('')
        
        for j in range (len(A)):
            for i in range (26):
            
                if (A[j]== alpha[i]):
                     
                    b+=(A[j]+str(i+1))
    
        return (b)
    

File: bacis test questions/char_index.java
//char index


public class Solution {
    public String solve(String A) {
        
       String alpha = ("abcdefghijklmnopqrstuvwxyz");
        String b=("");
        
        for(int j=0; j<A.length(); j++){
            for (int i=0; i<26; i++){
            
                if (A.charAt(j)== alpha.charAt(i)){
                     
                    b+=(A.charAt(j)+Integer.toString(i+1));
                }
            }
        }
    return (b);
    }
}